test/unit/models/user.spec.ts

```diff
@@ -11,12 +17,18 @@ describe("Models.User", () => {
   let clock: sinon.SinonFakeTimers;
   let today: Date;
   let user: IUser;
+  let db: Model<IUserModel>;
 
   beforeEach(() => {
+    container.unbind(TYPES.MongoConnector);
```

Comment:
why unbind?  If its to reset something shouldnt it be called in an afterEach() ?